portfolio/src/main/java/com/google/sps/servlets/HomeServlet.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/login")
+public class HomeServlet extends HttpServlet {
+  private static final String HOME = "/";
```

Comment:
Can you make the variable name more descriptive? I do not know what HOME means in terms of "/". Is it HOME_PATH, ...

Reply:
Yes, it is referring to the home page.

Done!